clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -140,51 +159,59 @@ func (s *schedulerLatencyMeasurement) getSchedulingLatency(c clientset.Interface
 		return nil, err
 	}
 
-	e2eSchedulingMetricHist := measurementutil.NewHistogram(nil)
+	e2eSchedulingDurationHist := measurementutil.NewHistogram(nil)
 	schedulingAlgorithmDurationHist := measurementutil.NewHistogram(nil)
-	for _, sample := range samples {
-		if sample.Metric[model.MetricNameLabel] == e2eSchedulingDurationMetricName {
-			measurementutil.ConvertSampleToHistogram(sample, e2eSchedulingMetricHist)
-			continue
-		}
-		if sample.Metric[model.MetricNameLabel] == schedulingAlgorithmDurationMetricName {
-			measurementutil.ConvertSampleToHistogram(sample, schedulingAlgorithmDurationHist)
-			continue
-		}
-		if sample.Metric[model.MetricNameLabel] != schedulingLatencyMetricName {
-			continue
-		}
 
-		var metric *measurementutil.LatencyMetric
-		switch sample.Metric[schedulermetric.OperationLabel] {
-		case schedulermetric.PredicateEvaluation:
-			metric = &result.PredicateEvaluationLatency
-		case schedulermetric.PriorityEvaluation:
-			metric = &result.PriorityEvaluationLatency
-		case schedulermetric.PreemptionEvaluation:
-			metric = &result.PreemptionEvaluationLatency
-		case schedulermetric.Binding:
-			metric = &result.BindingLatency
-		}
```

Comment:
in a follow up PR, we should add the histogram based metrics for preemption and binding: 

Preemption: https://github.com/kubernetes/kubernetes/blob/60df45fa55746fa2e9c9c116d09c5280cba9059f/pkg/scheduler/metrics/metrics.go#L118

binding: https://github.com/kubernetes/kubernetes/blob/60df45fa55746fa2e9c9c116d09c5280cba9059f/pkg/scheduler/metrics/metrics.go#L127